modules/ossm-about-collecting-ossm-data.adoc

```diff
@@ -13,5 +13,12 @@ To collect {ProductName} data with `must-gather`, you must specify the {ProductN
 
 [source,terminal]
 ----
-$ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel7
+$ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8
+----
+
+To collect {ProductName} data for a specific namespace with `must-gather`, you must specify the {ProductName} image and namespace. In this example, replace  `<namespace>` with your namespace.
```

Comment:
"for a specific control plane namespace" and the other part of the paragraph "... replace `<namespace>` with your control plane namespace.

Reply:
@neal-timpe if you have questions about what I mentioned above, please check this on maistra io docs: https://github.com/maistra/maistra.github.io/pull/176